judge/bridge/judgecallback.py

```diff
@@ -178,6 +178,8 @@ def on_grading_end(self, packet):
         submission.case_total = total
 
         sub_points = round(points / total * submission.problem.points if total > 0 else 0, 3)
+        if submission.user.is_unlisted:
```

Comment:
Not what I meant. I meant whether it's actually a good idea to do it here, basically giving a checkbox to make a users submission worth 0 point for the entire duration of the box being checked. Other methods may allow such users to regain points, or get no points for free by just checking the box, while this requires rescoring, which would not have worked anyways since rescore doesn't use this code.